tests/v2/test_jsonrpc_waku.nim

```diff
@@ -106,6 +106,77 @@ procSuite "Waku v2 JSON-RPC API":
     server.stop()
     server.close()
     waitfor node.stop()
+  
+  asyncTest "relay_polling": 
```

Comment:
Can we have a more descriptive name of what this is testing?

Reply:
Will do!